### ISSUE TITLE
Doesn't work with create-react-app

### ISSUE THREAD

**Issue by corysimmons** [nathanhleung/install-peerdeps#63]
`$ npx install-peerdeps ookook`

I created a create-react-app project and tried to use install-peerdeps to install another package, and its peers, to the CRA app.

```bash
npx create-react-app foo
cd foo
npx install-peerdeps ookook
```

[I have peerdeps](https://github.com/corysimmons/ookook/blob/294ade30abde80066feec98d89a9abb189c721d5/package.json#L23-L29) in that file.

What am I doing wrong? :\

Also, I've had problems with understanding how this project works exactly in the past. High level, how is it supposed to work? I would assume it would download a copy of `ookook` from npm, then look at the package.json of it for a peerDeps field, then loop over those deps running `npm/yarn install/add dep`.

**Comment by corysimmons** [comment]
Huh. Works as expected without CRA...

Guess I should rename this issue.

**Comment by corysimmons** [comment]
Seems to be related to `It seems as if you are using Yarn. Would you like to use Yarn for the installation? (y/n) y`

If I enter anything there, it just exits out silently/immediately. But `npx install-peerdeps ookook -Y` works as expected.

**Comment by AllanFerencz** [comment]
I'm currently having the same issue with a different package.

npx install-peerdeps --dev eslint-config-wesbos

With a Next.js project.
I'm going to try passing the Y flag.

**Comment by nathanhleung** [comment]
Thanks for the report.

> Also, I've had problems with understanding how this project works exactly in the past. High level, how is it supposed to work? I would assume it would download a copy of `ookook` from npm, then look at the package.json of it for a peerDeps field, then loop over those deps running `npm/yarn install/add dep`.

The project pulls the desired package's package.json from the registry, and then installs the desired package + all the peer dependencies in one command.

I'll add a note to the README to use the `-Y` flag with `create-react-app`.

**Comment by vhakulinen** [comment]
> If I enter anything there, it just exits out silently/immediately. But npx install-peerdeps ookook -Y works as expected.

I had the same problem and this worked.

**Comment by corysimmons** [comment]
I think the solution is to fix this inside the install-peerdeps lib rather than just adding a note to README.

**Comment by nathanhleung** [comment]
I think this should be fixed by #126. Please reopen with the command you ran and logs (if any) if this  issue persists.
